src/demo-app/a11y/a11y.html

```diff
@@ -2,11 +2,12 @@
 
 <nav>
   <a (click)="skipNavigation()" href="javascript:void(0)">Skip navigation</a>
-  <a *ngFor="let navItem of navItems" [routerLink]="[navItem.route]">
+  <a *ngFor="let navItem of navItems" [routerLink]="[navItem.route]" (click)="skipNavigation()">
     {{navItem.name}}
   </a>
 </nav>
 
 <div #maincontent tabindex="0">
+  <h1 #header tabindex="0" *ngIf="!!currentComponent">{{currentComponent}} Demo</h1>
```

Comment:
We should make the `tabindex="-1"` so that it can be focused programmatically without being in the tab order